vite-project/src/model/CartItem.js

```diff
@@ -1,7 +1,10 @@
-import getId from '../utils/getId';
+import getId from "../utils/getId";
 
 class CartItem {
-
+  constructor(name, price) {
+    this.id = getId();
+    (this.name = name), (this.price = price);
```

Comment:
Is there a reason you did it this way? Why not like this:?

```suggestion
    this.name = name;
    this.price = price;
```